src/StatsBase.jl

```diff
@@ -189,53 +206,16 @@ module StatsBase
 
     ConvergenceException
 
-@static if !isdefined(Base, :midpoints)
-    export midpoints
-end
-
-const BASESTATS_IN_STATSBASE = VERSION >= v"0.7.0-DEV.5238"
-
-if VERSION < v"0.7.0-DEV.3665"
-    myscale!(A::AbstractArray, b::Number) = scale!(A, b)
-else
-    myscale!(A::AbstractArray, b::Number) = rmul!(A, b)
-end
+const BASESTATS_IN_STATSBASE = v"0.7.0-DEV.5238" <= VERSION < v"0.7.0-beta.85"
```

Comment:
Since that variable is only used here, better put it directly with the `if` so that Femtocleaner can remove it.

Reply:
It's used in a few other places as well:
```
$ sift -e BASESTATS
test/runtests.jl:28:if StatsBase.BASESTATS_IN_STATSBASE
src/StatsBase.jl:209:const BASESTATS_IN_STATSBASE = v"0.7.0-DEV.5238" <= VERSION < v"0.7.0-beta.85"
src/StatsBase.jl:211:@static if BASESTATS_IN_STATSBASE
```